File: Python/Problem_1.py
import math
n = int(input("Enter the number of food items ordered (n): "))
count = 0
delivery_times = []

print("Enter the delivery times of food items")
for i in range(n):
    el = int(input())
    delivery_times.append(el)
    if el <= 30:
        count += 1

if count >= math.floor(n / 2):
    print("Satisfied")
else:
    print("Frustrated")

File: Java/Problem_1.java
import java.util.*;
public class Problem_1{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        System.out.print("Enter the number of food items ordered (n): ");
        int el;
        int count=0;
        List<Integer>delivery_times=new ArrayList<>(n);
        System.out.println("Enter the delivery times of food items");
        for(int i=0;i<n;i++){
            el=sc.nextInt();
            delivery_times.add(el);
            if(el<=30) count++;
        }
        if(count >= n/2) System.out.println("Satisfied");
        else System.out.println("Frustrated");
        sc.close();
    }
}